### PR TITLE
fix(binding-mqtt): fix eslint errors & warnings

### PR DESCRIPTION
**Pull request by danielpeintner**

### CODE REVIEW

File: packages/binding-mqtt/src/mqtt-client.ts [hunk 1]
```diff
@@ -37,24 +37,24 @@ export default class MqttClient implements ProtocolClient {
...
-            let retain = form["mqtt:retain"]; // TODO: is this needed here?
-            let qos = form["mqtt:qos"]; // TODO: is this needed here?
-            let requestUri = url.parse(form["href"]);
-            let topic = requestUri.pathname.slice(1);
-            let brokerUri: String = "mqtt://" + requestUri.host;
-
-            if (this.client == undefined) {
+            const contentType = form.contentType;
+            // const retain = form["mqtt:retain"]; // TODO: is this needed here?
+            // const qos = form["mqtt:qos"]; // TODO: is this needed here?
```
Comment (relu91) [inline packages/binding-mqtt/src/mqtt-client.ts:52]: I would get rid of those, if we'll need to re-introduce them we can. We can always use git to keep track of old code.

File: packages/binding-mqtt/src/mqtt.ts [hunk 2]
```diff
@@ -43,8 +43,8 @@ export enum MqttQoS {
 
 export class MqttForm extends Form {
     public "mqtt:qos": MqttQoS = MqttQoS.QoS0;
-    //public 'mqtt:topic': string = ''
-    public "mqtt:retain": Boolean;
+    // public 'mqtt:topic': string = ''
```
Comment (relu91) [inline packages/binding-mqtt/src/mqtt.ts:46]: Remove?

File: packages/binding-mqtt/src/mqtt-client.ts (hunk 1 above)
Comment (danielpeintner) [reply packages/binding-mqtt/src/mqtt-client.ts]: removed commented out code

File: packages/binding-mqtt/src/mqtt.ts (hunk 2 above)
Comment (danielpeintner) [reply packages/binding-mqtt/src/mqtt.ts]: removed commented out code

**Review by relu91 — APPROVED** [review]

**Pull request merged**